packages/@react-aria/test-utils/src/tree.ts

```diff
@@ -137,7 +136,7 @@ export class TreeTester {
     // this would be better than the check to do nothing in events.ts
     // also, it'd be good to be able to trigger selection on the row instead of having to go to the checkbox directly
     if (interactionType === 'keyboard' && (!checkboxSelection || !rowCheckbox)) {
-      await this.keyboardNavigateToRow({row, useAltKey: selectionBehavior === 'replace'});
+      await this.keyboardNavigateToRow({row, avoidSelectionOnNav: selectionBehavior === 'replace'});
```

Comment:
"avoid" feels a bit, "i'll try my best, but it might happen" to me, maybe one of these instead?
`noSelectionOnNav`
`shouldNotSelectOnNav`

Reply:
haha, I was thinking about exactly this when naming the option. I actually did want to give that kind of impression since using `noSelectionOnNav` or `shouldNotSelectOnNav` would mean it would sound like selection SHOULD change on nav if those were false, something which wouldn't happen for a collection with `selectionBehavior: 'toggle'` without me firing Enter after every ArrowUp/Down.

`avoidSelectionOnNav` is more of the following: if `true` we endeavor to avoid selection on keyboard navigation via the Alt modifier, otherwise if `false` we won't do anything special when keyboard navigating and if something gets selected as you keyboard navigate that is expected due to the `selectionBehavior` of your collection

Reply:
hrmmmmm..... sounds tricky to document. maybe instead of a boolean, it should be
`selectionOnNav: 'default' | 'none'` and it defaults to `'default'`

Reply:
that sounds good, will update